_data/people.yaml

```diff
@@ -225,6 +225,21 @@ alexandra-holinski:
   last-name: Holinski
   orcid: 0000-0003-2321-5568
   pronouns: She/her
+LAlmehlisy:
```

Comment:
Could you move Leena below in the list? We try to use alphabetical order on the username :)

Reply:
Can you also put the username in lowercase? Thanks a lot

Reply:
I moved it below with other usernames starting with L